definitions/3.0.0/components.json

```diff
@@ -68,10 +68,34 @@
         "$ref": "http://asyncapi.com/definitions/3.0.0/messageTrait.json"
       }
     },
-    "serverBindings": {
```

Comment:
Why are you removing the `serverBindings`? 🤔

Reply:
LOL, good catch! 👀